stylus/include/WhereToFindUs.styl

```diff
@@ -0,0 +1,51 @@
+.WhereToFindUs
+	margin-bottom: 3rem
+	margin-top: 5rem
+	text-align: center
+	>h1
+		font-family: Arvo
+		margin-bottom: 3rem
+	>p
+		margin-right: 33%
+		margin-left: 33%
+	>div:nth-child(3)
+		display: flex
+		flex-wrap: wrap
+		margin-left: 8rem
+		margin-top: 5rem
+		text-align: left
+		>div
+			flex: 1
+			flex-basis: 25rem
+			margin-bottom: 4rem
+			>h3
+				font-weight: bold
+				margin-bottom: 1rem
+			>p
+				margin-bottom: 0rem
+			>p:nth-child(3)
+				margin-bottom: 1rem
+	>div:nth-child(4)
```

Comment:
@cbaron I think this arrangement of images occurs enough in the site to justify a refactor of the styles (probably move everything to `base.styl`). What do you think? It's going to be on nearly every page it seems, so might as well clean it up.

Reply:
I was thinking the images need to be there own view that is dumped on each page. 

Basically we want folks to be able to quickly reference recent things we've done.  Eventually it will be a single API call to get the x most recent items ( blog posts, recipes, whatever )

Reply:
Ok probably best to hold off for now then.